yt_idefix/fields.py

```diff
@@ -47,3 +62,144 @@ def setup_fluid_fields(self):
         setup_magnetic_field_aliases(
             self, "idefix-dmp", [f"Vc-BX{idir}" for idir in "123"]
         )
+        for idir in "123":
+            self.alias(
+                ("gas", f"Vc-BX{idir}"),
+                ("idefix-dmp", f"Vc-BX{idir}"),
+                units="code_magnetic",
+            )
+
+
+class PlutoXdmfFields(FieldInfoContainer):
```

Comment:
@dutta-alankar @neutrinoceros The field names of Pluto-vtk and Pluto-xdmf should be the same (not sure about Idefix), so I suggest to use a common class `PlutoField` instead.

Reply:
Good point! However, the names of some of the fields are a bit different in `.vtk` and `.h5` files. The differences are not significant (like `BX1` in `.vtk` is `Bx1` in `.h5`) and perhaps can be merged together.

Reply:
since field names are slightly different, we cannot fuse these classes. Otherwise, I agree with @xshaokun that we should aim for maximal similarity. This is what yt standardisation is supposed to provide: a common interface that shows little to no difference in usage with respect to the actual data format. For the record this is why I'm advocating against custom field aliases in the `"gas"` field type.

Reply:
> Also the tracer counting logic needs to be put in for `.vtk` files.

@dutta-alankar Could you explain why we need the tracer counting part?

> The differences are not large (like `BX1` in `.vtk` is `Bx1` in `.h5`)

As far as I know, the differences are introduced in this frontend instead of in Pluto (refer to the comment in `vtk_io.read_field_offset_index()`). The variable names of PlutoVtkDataset are uppercased before to avoid duplicating field names. As for now, we should undo this uppercase operation to avoid duplicating Pluto field class. Do you agree? @neutrinoceros